.github/workflows/build-plugin/action.yml

```diff
@@ -0,0 +1,21 @@
+name: Build Plugin
+description: Inject the given version into the plugin files and then build it and zip it.
+
+inputs:
+  PACKAGE_VERSION:
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - run: |
+            sed -i -E "s/Stable tag: .*/Stable tag: ${PACKAGE_VERSION}/g" ./readme.txt
+            sed -i -E "s/Version: .*/Version: ${PACKAGE_VERSION}/g" elementor.php
+            sed -i -E "s/ELEMENTOR_VERSION', '.*'/ELEMENTOR_VERSION', '${PACKAGE_VERSION}'/g" elementor.php
+            PLUGIN_ZIP_FILENAME="elementor-${PACKAGE_VERSION}.zip"
+            npx grunt build
+            mv build elementor
+            zip -r $PLUGIN_ZIP_FILENAME elementor
+            echo "PLUGIN_ZIP_FILENAME=${PLUGIN_ZIP_FILENAME}" >> $GITHUB_ENV
+            echo "PLUGIN_ZIP_FILENAME=${PLUGIN_ZIP_FILENAME}"
+      shell: bash
```

Comment:
Move before the run

Reply:
If the "shell: bash" will be before the "run" the action will not work.
As the docs also mentioned, I'm getting also warning on my IDE.

https://docs.github.com/en/actions/creating-actions/creating-a-composite-action